cartography/data/jobs/cleanup/gcp_sql_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
GCPSQLUser deletion should happen first and then the instance. Otherwise, the relationship based query may not return any data.